spec/proto/basic.spec.coffee

```diff
@@ -27,11 +27,11 @@ describe "Basic setup", ->
     expect($F(select)).toBe ""
 
     container = div.down(".chosen-container")
-    container.simulate("mousedown") # open the drop
+    Chosen.trigger(container, "mousedown") # open the drop
```

Comment:
this is not a good idea, because ``Chosen.trigger`` is not a full-fledged implementation of triggering builtin events (it always relies on the Event class, while many DOM events are using subclasses with extra properties)